products/sle-micro/main.pm

```diff
@@ -24,6 +27,20 @@ unregister_needle_tags('ENV-OFW-1');
 unregister_needle_tags('ENV-PXEBOOT-0');
 unregister_needle_tags('ENV-PXEBOOT-1');
 
+sub load_boot_from_disk_tests {
+    if (is_s390x()) {
+        loadtest 'installation/bootloader_start';
+        loadtest 'boot/boot_to_desktop';
+    } else {
+        loadtest 'microos/disk_boot';
+    }
+    loadtest 'transactional/host_config';
+    loadtest 'console/suseconnect_scc';
+    loadtest 'transactional/enable_selinux' if get_var('ENABLE_SELINUX');
+    loadtest 'transactional/install_updates' if is_released;
```

Comment:
neat!

Reply:
is_released contains the condition FLAVOR='MicroOS-Image-Updates'?

Reply:
It is a regex that matches last part of the flavor.

```
=head2 is_released

Returns true if called on a released build
=cut
sub is_released {
    return get_var('FLAVOR') =~ /Incidents|Updates|QR/;
}
```